assets/javascript/game.js

```diff
@@ -0,0 +1,106 @@
+//  a.	    Compare the “Your total score is” to the number that is random generated by the computer.
+    // i.	If your total score is less than the random generated number, allow the user to keep clicking crystals – and with each crystal clicked, the “Your total score is:” will increase by the number associated with the crystal.
+    // ii.	If “Your total score is:” is equal to the random generated number:
+        // 1.	Increment “wins” by 1 (var wins++1)
+        // 2.	Reset “Your total score is:” to blank 
+        // 3.	Generate a new random number to be displayed and used to compare the “Your total score is:” to
+//      iii.	If “Your total score is:” is greater than the random generated number:
+        // 1.	Increment “losses” by 1 (var wins++1)
+        // 2.	Reset “Your total score is:” to blank 
+        // 3.	Generate a new random number to be displayed and used to compare the “Your total score is:” to.
+
+//Begin Javascript and jQuery
+
+<script type="text/javascript">
+// Creates an array that computer chooses and adds them to the array.
+
+var computerChoice = ["18", "36", "72", "90", "108", "120", "24", "48", "72", "96"];
+
+var crystalValues = [2, 8, 4, 18]
+
+for (var i = 0; i < crystalValues.length; i++) {
+
+
+// Creating variables to hold the number of wins, losses, and total score.
+
+var wins = 0;
+var losses = 0;
+var totalScore = 0;
+
+// Create variables that hold references to the places in the HTML where we want to display things.
+var randomNumberText = document.getElementById("randomnumber-text");
+var computerChoiceText = document.getElementById("computerchoice-text");
+var winsText = document.getElementById("wins-text");
+var lossesText = document.getElementById("losses-text");
+var userChoiceText = document.getElementById("usernchoicetext");
+var totalScoreText = document.getElementById("totalscore-text");
+
+
+
+// This function is run whenever the user presses a key.
+    document.onkeyup = function(event) {
+
+    // Determine score so far.
+    var totalScore = event.key;
+
+
+    console.log(userChoice)
+
+    // Randomly chooses a choice from the options array. This is the Computer's guess.
+    var totalScore = computerChoice[Math.floor(Math.random() * computerChoice.length)];
+
+    // If the choices match, increment our wins variable and reset total score so far to 0.
+    if (totalScore === computerGuess) {
+        wins++;
+        totalScore = 0
+    }
+        
+        else if (totalScore !== computerGuess) {
+        losses++;
+        totalScore = 0
+        }
+
+        if (totalScore === 0) {losses++; computerGuess="";
+        }
+
+    // Display the user guesses, and wins/losses.
+    userChoiceText.textContent = "You chose: " + userChoice;
+    computerChoiceText.textContent = "The computer chose: " + computerGuess;
+    winsText.textContent = "wins: " + wins;
+    lossesText.textContent = "losses: " + losses;
+    totalScoreText.textContent = "total score:" + totalScore;
+    }
```

Comment:
if you decide to come back to this homework assignment, please let me know! I'm willing to help you out :)